go.mod

```diff
@@ -7,51 +7,76 @@ require (
 	github.com/Khan/genqlient v0.5.0
 	github.com/bhoriuchi/graphql-go-tools v1.0.0
 	github.com/containerd/containerd v1.6.8
-	github.com/docker/distribution v2.8.1+incompatible
+	github.com/containerd/fuse-overlayfs-snapshotter v1.0.2
+	github.com/containerd/stargz-snapshotter v0.12.0
+	github.com/coreos/go-systemd/v22 v22.3.3-0.20220203105225-a9a7ef127534
+	github.com/docker/cli v20.10.17+incompatible
+	github.com/docker/docker v20.10.17+incompatible
 	github.com/go-openapi/runtime v0.24.1
 	github.com/gofrs/flock v0.8.1
 	github.com/graphql-go/graphql v0.8.0
 	github.com/graphql-go/handler v0.2.3
+	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/moby/buildkit v0.10.4
+	github.com/moby/buildkit v0.10.1-0.20220912044125-d4bc7980aa57
```

Comment:
This is okay for now because it was easier to get going, but I think we should try to track official releases of buildkit for the time being (so `v0.10.4`). I know that will probably be an annoying update to make, so don't worry about it immediately, but we should at least make a follow-up for this.